assets/src/edit-story/components/form/color/colorInput.js

```diff
@@ -209,9 +214,11 @@ const ColorInput = forwardRef(function ColorInput(
         isOpen={pickerOpen}
         placement={PLACEMENT.LEFT_START}
         spacing={spacing}
+        invisible={eyedropperActive}
```

Comment:
There should probably be a way to get out of the eyedropper state without actually choosing the colour.

Reply:
Yes, that's on the list (Support all cases for closing the eyedropper), so click outside the canvas and ESC.